src/main/webapp/data/userMapData.json

```diff
@@ -436,7 +436,6 @@
       "HKUST",
       "Queen Elizabeth School",
       "The Chinese University of Hong Kong",
-      "The Education University of Hong Kong",
```

Comment:
Actually, I meant to remove the other one, since the university brands itself with `The` but I guess it doesn't really matter.